server/controllers/reviews.js

```diff
@@ -0,0 +1,72 @@
+const Profile = require("../models/Profile");
+const Request = require("../models/Request");
+const Review = require("../models/Review");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /reviews/
+// @desc Post new review for a profile
+// @access Private
+exports.postReview = asyncHandler(async (req, res, next) => {
+  const { reviewedProfileId, requestId, starRating, text } = req.body;
```

Comment:
Validate request body and return a `400` if any of these is not provided

Reply:
@ezeilo-su All of these fields are getting validated in a separate `validatePostReview` function with the express-validator.